Habitica/src/main/java/com/habitrpg/android/habitica/ui/activities/TaskFormActivity.java

```diff
@@ -420,7 +480,215 @@ public void onKeyboardClose() {
+            });
+            repeatablesWeekDayCheckboxes.add(checkbox);
+            repeatablesFrequencyContainer.addView(weekdayRow);
+        }
+
+        generateSummary();
+    }
+
+    private void generateSummary () {
+        String frequency = repeatablesFrequencySpinner.getSelectedItem().toString();
+        String everyX = String.valueOf(repeatablesEveryXSpinner.getValue());
+        String frequencyQualifier = "";
+
+        switch (frequency) {
+            case "Daily":
+                frequencyQualifier = "day(s)";
+                break;
+            case "Weekly":
+                frequencyQualifier = "week(s)";
+                break;
+            case "Monthly":
+                frequencyQualifier = "month(s)";
+                break;
+            case "Yearly":
+                frequencyQualifier = "year(s)";
+                break;
+        }
+
+        String weekdays = "";
+        List<String> weekdayStrings = new ArrayList<>();
+        int offset = firstDayOfTheWeekHelper.getDailyTaskFormOffset();
+        if (this.repeatablesWeekDayCheckboxes.get(offset).isChecked()) {
+            weekdayStrings.add("Monday");
+        }
+        if (this.repeatablesWeekDayCheckboxes.get((offset + 1) % 7).isChecked()) {
+            weekdayStrings.add("Tuesday");
+        }
+        if (this.repeatablesWeekDayCheckboxes.get((offset + 2) % 7).isChecked()) {
+            weekdayStrings.add("Wednesday");
+        }
+        if (this.repeatablesWeekDayCheckboxes.get((offset + 3) % 7).isChecked()) {
+            weekdayStrings.add("Thursday");
+        }
+        if (this.repeatablesWeekDayCheckboxes.get((offset + 4) % 7).isChecked()) {
+            weekdayStrings.add("Friday");
+        }
+        if (this.repeatablesWeekDayCheckboxes.get((offset + 5) % 7).isChecked()) {
+            weekdayStrings.add("Saturday");
+        }
+        if (this.repeatablesWeekDayCheckboxes.get((offset + 6) % 7).isChecked()) {
+            weekdayStrings.add("Sunday");
+        }
+        weekdays = " on " + TextUtils.join(", ", weekdayStrings);
+        if (!frequency.equals("Weekly")) {
+            weekdays = "";
+        }
+
+        if (frequency.equals("Monthly")) {
+            weekdays = "";
+            Calendar calendar = startDateListener.getCalendar();
+            String monthlyFreq = repeatablesOnSpinner.getSelectedItem().toString();
+            if (monthlyFreq.equals("Day of Month")) {
+                Integer date = calendar.get(Calendar.DATE);
+                weekdays = " on the " + date.toString();
+            } else {
+                Integer week = calendar.get(Calendar.WEEK_OF_MONTH);
+                String dayLongName = calendar.getDisplayName(Calendar.DAY_OF_WEEK, Calendar.LONG, Locale.getDefault());
+                weekdays = " on the " + week.toString() + " week on " + dayLongName;
+            }
+        }
+
+        String summary = "Repeats " + frequency + " every " + everyX + " " + frequencyQualifier + weekdays;
```

Comment:
Yes, they can. `This %1$s a %2$s string`  per example for two strings. https://developer.android.com/guide/topics/resources/string-resource.html#FormattingAndStyling